dwave_networkx/drawing/qubit_layout.py

```diff
@@ -60,6 +63,16 @@ def draw_qubit_graph(G, layout, linear_biases={}, quadratic_biases={},
         be. If not provided, the colormap will default to the middle of
         min/max values provided.
 
+    line_plot : boolean (optional, default False)
```

Comment:
I'm encountering some problems with this parameter: 

1. "In this drawing style, the interpretation the width and node_size parameters (provided in kwargs) determines..." 

I think anything in ``kwargs`` should be optional, but not providing ``width`` and ``node_size`` parameters will cause an error: ``TypeError: draw_lineplot() missing 2 required keyword-only arguments: 'node_size' and 'width'``

2. The function fails for what seems to me simple input:
```
>>> import dwave_networkx as dnx
>>> G = dnx.chimera_graph(1)
>>> layout = dnx.chimera_layout(G)
>>> draw_qubit_graph(G, layout, line_plot=True, node_size=2, width=2)
```
Fails with ``TypeError: cannot unpack non-iterable numpy.float64 object``

Reply:
This function isn't really meant to be called by users directly (note that it is not in the root `dwave_networkx` namespace).  The following should work:

```python
import dwave_networkx as dnx
G = dnx.chimera_graph(1)
layout = dnx.chimera_layout(G, plot_kwargs={'line_plot': True})
dnx.drawing.qubit_layout.draw_qubit_graph(G, layout, line_plot=True, node_size=2, width=2)
```

Reply:
This still fails but with the problem [noted here](https://github.com/dwavesystems/dwave-networkx/pull/214#discussion_r1089351957):

```
>>> dnx.chimera_layout(G, plot_kwargs={'line_plot': True})
...
TypeError: '_AxesStack' object is not callable
```

Reply:
yeah, I'm working on that

Reply:
If you'd like; try it with matplotlib 3.4.1

Reply:
yes, that works, thanks